### PR TITLE
Run Flow

### PR DESCRIPTION
**Pull request by waigani**
Use the generic Run flow to kick off Flows.

Note: this relies on this PR: https://github.com/codelingo/rpc/pull/5

### CODE REVIEW

File: app/commands/codemod/review.go [hunk 1]
```diff
@@ -61,12 +64,51 @@ func RequestReview(ctx context.Context, req *flow.ReviewRequest) (chan *flow.Iss
...
+				errc <- err
+				continue
+			}
+
+			issuec <- issue
+		}
+		wg.Done()
+	}()
+
+	go func() {
```
Comment (emersonwood) [inline app/commands/codemod/review.go]: Could replace the 3 go routines and wait group with a single routine and a for select loop.

File: app/commands/review/review.go [hunk 2]
```diff
@@ -30,12 +33,51 @@ func RequestReview(ctx context.Context, req *flow.ReviewRequest) (chan *flow.Iss
...
+				errc <- err
+				continue
+			}
+
+			issuec <- issue
+		}
+		wg.Done()
+	}()
+
+	go func() {
```
Comment (emersonwood) [inline app/commands/review/review.go]: Same as above, could replace the 3 go routines and wait group with a single routine and a for select loop.

File: app/commands/codemod/review.go [hunk 3]
```diff
@@ -61,12 +64,51 @@ func RequestReview(ctx context.Context, req *flow.ReviewRequest) (chan *flow.Iss
...
 		return nil, nil, errors.Trace(err)
 	}
 
-	return issuec, errorc, nil
+	issuec := make(chan *flow.Issue)
+	errc := make(chan error)
+	wg := sync.WaitGroup{}
+	wg.Add(2)
+
+	go func() {
```
[the remark this answers is not in the document]
Comment (BlakeMScurr) [reply app/commands/codemod/review.go]: @CodeLingoBot ignore

[the remark this answers is not in the document]
Comment (BlakeMScurr) [reply app/commands/codemod/review.go]: @CodeLingoBot ignore

[the remark this answers is not in the document]
Comment (BlakeMScurr) [reply app/commands/codemod/review.go]: @CodeLingoBot ignore

[the remark this answers is not in the document]
Comment (BlakeMScurr) [reply app/commands/codemod/review.go]: @CodeLingoBot ignore

**Pull request merged**